tests/test_reader.py

```diff
@@ -269,8 +257,7 @@ def test_regex_escape():
 
 
 @pytest.mark.parametrize('root', ['pc', 'gd'])
-@pytest.mark.skipif(pytables_imported is False,
-                    reason="requires tables package")
+@pytables_mark_xfail
```

Comment:
Should this test be simply skipped rather than xfailed?

Reply:
If we don't run it, then we miss 100% coverage -- one could write specific tests which check for failure, but I think this is neater.